pkg/webui/console/views/application-access-edit/index.js

```diff
@@ -119,23 +173,24 @@ export default class ApplicationAccessEdit extends React.Component {
     )
 
     const initialFormValues = {
+      id: apiKey.id,
       name: apiKey.name,
-      description: apiKey.description,
-      ...rightsValues,
+      rights: { ...rightsValues },
     }
 
     return (
       <Container>
         <Row>
           <Col lg={8} md={12}>
+            <IntlHelmet title={m.accessEdit} values={{ keyId }} />
```

Comment:
```suggestion
            <IntlHelmet title={m.accessEdit} values={{ keyName || 'Access Key' }} />
```
Let's not use the key id in the title, it's just an unhelpful, lengthy salad of letters. I'd suggest to use the key name and falling back to a generic 'Access Key'.